code/modules/projectiles/guns/energy/laser/carpediem.dm

```diff
@@ -5,19 +5,19 @@
 	item_state = null	//so the human update icon uses the icon_state instead.
 	item_charge_meter = TRUE
 	desc = "Created by the Numericals and distributed by the Testament in hopes of combatting Excelsior and psionic creatures. \
-	The Carpediem Musket is a laser charge rifle designed for ceremonial usage, though it also packs an impressive punch- even if its rate of fire leaves a lot to be desired."
+	The Carpediem Musket is a laser musket rifle designed for ceremonial usage, though it also packs an impressive punch- even if its rate of fire leaves a lot to be desired."
 	fire_sound = 'sound/weapons/energy/lasmusket_fire.ogg'
 	slot_flags = SLOT_BACK
 	w_class = ITEM_SIZE_BULKY
-	matter = list(MATERIAL_PLASTEEL = 5, MATERIAL_WOOD = 10, MATERIAL_GLASS = 3, MATERIAL_SILVER = 2, MATERIAL_GOLD = 1)
+	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_WOOD = 10, MATERIAL_GLASS = 3, MATERIAL_SILVER = 3, MATERIAL_GOLD = 3, MATERIAL_BIOMATTER = 15)
 	projectile_type = /obj/item/projectile/beam/musket
-	fire_delay = 20 //Musket so you know
-	charge_cost = 125 // 4 shots with some change on a 600M
+	fire_delay = 100 //the original weapon used to have a charge mode, to make it a zero sum, the delay had to be adjsuted
```

Comment:
```suggestion
	fire_delay = 20 //the original weapon used to have a charge mode, to make it a zero sum, the delay had to be adjsuted
```
Having to wait 10 seconds per shot is annoying for such a low power

Reply:
it was 20 with charge mode, charging itself is around 5 seconds, perhaps 70 since the charge mode was lost?

Reply:
also take into account the overall balance of the gun